manifests/apps/jupyter/deploy-python-model-quickstart.yaml

```diff
@@ -9,61 +9,61 @@ spec:
   appName: jupyterhub
   durationMinutes: 10
   icon: 'images/jupyterhub.svg'
-  description: How to deploy a Python model using Flask and OpenShift.
+  description: Deploy a Python model by using Flask and OpenShift
```

Comment:
@Jooho @andrewballantyne is this method of deploying models with Flask still a good idea to show, since we now have model serving / model mesh?